src/ripple/rpc/handlers/Submit.cpp

```diff
@@ -117,12 +117,11 @@ doSubmit(RPC::JsonContext& context)
         }
     }
 
-    std::string reason;
-    auto tpTrans = std::make_shared<Transaction>(stpTrans, reason, context.app);
+    auto tpTrans = std::make_shared<Transaction>(stpTrans);
     if (tpTrans->getStatus() != NEW)
     {
         jvResult[jss::error] = "invalidTransaction";
-        jvResult[jss::error_exception] = "fails local checks: " + reason;
+        jvResult[jss::error_exception] = "fails local checks";
 
         return jvResult;
     }
```

Comment:
This block should be impossible to reach now. It can probably be safely replaced with `assert(tpTrans->getStatus() == NEW);`